composer.json

```diff
@@ -89,28 +89,27 @@
     }
   },
   "require": {
-    "composer/installers": "^1.6",
-    "cweagans/composer-patches": "~1.0",
+    "composer/installers": "^1.12",
+    "cweagans/composer-patches": "^1.7",
     "drupal/core-composer-scaffold": "^9.4",
     "drupal/core-project-message": "^9.4",
     "drupal/core-recommended": "^9.4",
-    "drupal/admin_toolbar": "^3.0",
-    "drupal/module_filter": "^3.2",
-    "drupal/cloud": "5.x-dev",
-    "drupal/ldap": "^4.0@beta",
-    "drupal/copyright_footer": "^1.7",
-    "drupal/login_destination": "^2.0@beta",
+    "drupal/admin_toolbar": "^3.1",
     "drupal/authorization": "^1.0@beta",
-    "drupal/externalauth": "^1.3",
-    "drupal/bootstrap": "^3.23",
+    "drupal/cloud": "5.x-dev",
     "drupal/cloud_orchestrator": "dev-5.x",
-    "drupal/panels": "^4.6",
+    "drupal/copyright_footer": "^2.0",
+    "drupal/ctools": "^3.7",
+    "drupal/externalauth": "^1.4",
+    "drupal/ldap": "^4.3",
+    "drupal/login_destination": "^2.0@beta",
+    "drupal/memcache": "^2.3",
+    "drupal/module_filter": "^3.2",
+    "drupal/page_load_progress": "^2.0",
     "drupal/page_manager": "^4.0@beta",
+    "drupal/panels": "^4.6",
```

Comment:
We can safely remove `drupal/page_manager` and `drupal/panels` now.  We are not using it anymore since we have our own dynamic homepage.